test/sagemaker_tests/tensorflow/tensorflow2_training/integration/conftest.py

```diff
@@ -198,6 +200,12 @@ def ecr_image(account_id, docker_base_name, tag, region):
     return "{}/{}:{}".format(registry, docker_base_name, tag)
 
 
+@pytest.fixture
+def sm_below_tf213_only(framework_version):
+    if Version(framework_version) in SpecifierSet(">=2.13"):
+        pytest.skip("Test only supports Tensorflow version below 2.13")
```

Comment:
Please double-check that this fixture works as intended by running the tests locally. You could change the actual test function to just `return` as soon as it enters, but the usage of this fixture should work as intended.

Reply:
Worked locally, but would like to run on the pipeline. will be fully ran on the SM PR as well.